core/server/common/src/main/java/alluxio/cli/Format.java

```diff
@@ -77,7 +84,7 @@ public static void main(String[] args) {
       System.exit(-1);
     }
     try {
-      format(args[0]);
+      format(Mode.valueOf(args[0]));
```

Comment:
I just tested and the error message changes to something like `java.lang.IllegalArgumentException: No enum constant alluxio.cli.Format.Mode.TEST`. We could catch exception here and print usage.